docs/docs/firmware/can-traffic/index.md

```diff
@@ -0,0 +1,81 @@
+
+# Calculating CAN traffic
+
+In order to calculate the CAN load, we must declare a few variables and their importance within the CAN protocol.
+
+## Baud
+
+The baud rate is the rate at which data is transfered in a communication channel (like CAN). For example, 9600 Baud rate is 9600 bits per second
+
+## Frequency
+
+The frequency of a message, expressed in (Hz), is how often a message is transmitted per second. Different messages can have different frequencies.
+
+This variation affects the total bits transmitted and, in turn, the bus load. To calculate bits per second, consider each message’s frequency and whether their lengths differ.
+
+## Message Length
+
+The message length in a CAN frame can be calculated as a function of the data length. This calculation includes several fixed fields and a variable data field, which changes based on the message's data content.
+
+<figure markdown="span">
```

Comment:
There is proper syntax for adding captions to an image but I didn't expect you to know it. I added it here for simplicity, just look it over. You can copy it from here if you ever need it.

Reply:
This section is a little confusing. There is a lot of bouncing around between sentences and the equations aren't consistent. For example, you say `b=8` and `length = a+bx` but then also say `M = 44 + 10x`. Is `b` 8 or 10?

I think it would be better to restructure this as a table:

--- 

The CAN frame length is made up of some fixed elements and a variable data length. Let $x$ be the number of bytes in the data section.

| Frame Element | Length (bits) |
|:---|:---|
|Start of Frame | 1 |
| Identifier | 11 |
| Remote Transmission Request | 1 |
| ... | ... |
| Data | 8 bits per byte + up to 20% stuffing = 9.6x |
| and so on |

The maximum total frame length is $1 + 11 + 1 + 9.6x = 44 + 9.6x$ (fill in the other elements of the equation)

Reply:
This will replace everything up to the Sample Calc header

Reply:
Would you like me to implement the syntax for adding captions. Would this be in regards to 
`Each CAN frame consists of several fields with fixed bit lengths, plus a variable-length data field. Let \(x\) be the number of bytes. Here’s the breakdown:` ?